src/System.Net.WebSockets.Client/tests/System.Net.WebSockets.Client.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{7C395A91-D955-444C-98BF-D3F809A56CE1}</ProjectGuid>
+    <OutputType>Library</OutputType>
+    <StringResourcesPath>..\src\Resources\Strings.resx</StringResourcesPath>
+    <UnsupportedPlatforms>Linux;OSX</UnsupportedPlatforms>
+  </PropertyGroup>
+
+  <!-- Help VS understand available configurations -->
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' " />
+
+  <ItemGroup>
+    <Compile Include="ClientWebSocketTests.cs" />
+    <Compile Include="ResourceHelper.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\src\System.Net.WebSockets.Client.csproj">
+      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
+      <OutputItemType>Content</OutputItemType>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Targets>Build;DebugSymbolsProjectOutputGroup</Targets>
+    </ProjectReference>
```

Comment:
This P2P reference looks like the kind we normally use for partial facade assemblies, which this is not.  Most of the corefx test projects use a simpler setup, e.g. https://github.com/dotnet/corefx/blob/master/src/System.ObjectModel/tests/System.ObjectModel.Tests.csproj#L44-L47